[8.2.2] Changelog (#146)
OntimizeWeb/ontimize-web-ngx-gallery: CHANGELOG.md

## 8.2.2(2023-11-28)
### Bug Fixes

> * Fixed bug in navigation main about image does not navigate to end ([15ff9e2](https://github.com/OntimizeWeb/ontimize-web-ngx-gallery/commit/15ff9e2)) Closes [#142](https://github.com/OntimizeWeb/ontimize-web-ngx-gallery/issues/142)

Reviewer: ```suggestion
* Fixed bug into navigation over main image that does not navigate to the last item([15ff9e2](https://github.com/OntimizeWeb/ontimize-web-ngx-gallery/commit/15ff9e2)) Closes [#142](https://github.com/OntimizeWeb/ontimize-web-ngx-gallery/issues/142)
```
